src/misc/extra/extraUtilUtil.c

```diff
@@ -304,8 +304,12 @@ char * Extra_UtilFileSearch(char *file, char *path, char *mode)
         return filename;
     }
     ABC_FREE(filename);
-    path = ++cp;
-    } while (! quit); 
+    if (quit) {
+        break;
+    } else {
+       path = ++cp;
```

Comment:
The same mistake fixed twice in this patch: the increment must not happen after the last round of the loop.